pkg/reconciler/events/scheduler/scheduler.go

```diff
@@ -187,7 +187,7 @@ func (r *Reconciler) deleteJob(ctx context.Context, scheduler *v1alpha1.CloudSch
 func (r *Reconciler) FinalizeKind(ctx context.Context, scheduler *v1alpha1.CloudSchedulerSource) reconciler.Event {
 	// If k8s ServiceAccount exists and it only has one ownerReference, remove the corresponding GCP ServiceAccount iam policy binding.
```

Comment:
this comment still applies? Don't recall if we agreed on removing the GSA, did we?
we might be removing a default one in some cases?

Reply:
This is referring to the default one. In the identity finalizer, it will do nothing if the KSA not pair to any GSA in the config-gcp-auth. I'll updating a more clear comment.

Reply:
ok, and by removal you mean removing the iam role binding, or whatever is called. Not the GSA itself, right?

Reply:
Yes, just remove the iam policy binding, not the GSA itself.